package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "tezster-gui-v2.0",
+  "name": "tezster-gui",
   "productName": "Tezster-GUI",
-  "version": "2.0.0",
-  "description": "Tezster-GUI",
+  "version": "2.1.0",
+  "description": "GUI version of Tezster which is ready to use Desktop app, made with React and Electron",
```

Comment:
Tezster GUI is a desktop app to interact & develop/deploy smart contracts on Tezos blockchain. 

you can consult Om for better one line description.